app/_hub/kong-inc/aws-lambda/1.0-x.md

```diff
@@ -0,0 +1,309 @@
+    - name: unhandled_status
+      required: false
+      default: "`200`, `202` or `204`"
+      description: |
+        The response status code to use (instead of the default `200`, `202`, or `204`) in the case of an [`Unhandled` Function Error](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseSyntax).
+    - name: forward_request_body
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the request body is to be sent in the `request_body` field of the JSON-encoded request. If the body arguments can be parsed, they will be sent in the separate `request_body_args` field of the request. The body arguments can be parsed for `application/json`, `application/x-www-form-urlencoded`, and `multipart/form-data` content types.
+    - name: forward_request_headers
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the original HTTP request headers are to be sent as a map in the `request_headers` field of the JSON-encoded request.
+    - name: forward_request_method
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the original HTTP request method verb is to be sent in the `request_method` field of the JSON-encoded request.
+    - name: forward_request_uri
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the original HTTP request URI is to be sent in the `request_uri` field of the JSON-encoded request. Request URI arguments (if any) will be sent in the separate `request_uri_args` field of the JSON body.
+    - name: is_proxy_integration
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the response format to receive from the Lambda to [this format](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-output-format). Note that the parameter `isBase64Encoded` is not implemented.
+
+  extra: |
+    **Reminder**: cURL by default sends payloads with an
+    `application/x-www-form-urlencoded` MIME type, which will naturally be URL-
+    decoded by Kong. To ensure special characters that are likely to appear in
+    your AWS key or secret (like `+`) are correctly decoded, you must
+    URL-encode them, hence use `--data-urlencode` if you are using cURL.
+    Alternatives to this approach would be to send your payload with a
+    different MIME type (like `application/json`), or to use a different HTTP client.
+
+---
+
+### Sending parameters
+
+Any form parameter sent along with the request will be also sent as an
+argument to the AWS Lambda function.
+
+### Notes
+
+If you do not provide an `aws.key` and `aws.secret`, the plugin uses an IAM role inherited from the instance running Kong.
```

Comment:
```suggestion
If you do not provide an `aws.key` and `aws.secret`, the plugin attempts to use an IAM role inherited from the instance running Kong.
```

Reply:
(id also suggest removing the extra newlines; keep this sentance and the next one paragraph